packages/components/addon/components/hds/pagination-bar/page-size.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="hds-pagination-page-size hds-typography-body-200 hds-font-weight-medium" ...attributes>
+  <label for={{this.pageSizeId}}>Items per page</label>
+  <Hds::Form::Select::Base id={{this.pageSizeId}} {{on "change" this.onChange}} @onChange={{@onChange}} as |S|>
```

Comment:
I see two "on change" handlers applied to the same element, probably we can unify them in a single call on the backing class. I can show you how if you want.